sdk/core/Azure.Core/tests/ClientDiagnosticsTests.Net50.cs

```diff
@@ -52,62 +68,49 @@ public void StartsActivitySourceActivity()
         {
             using var _ = SetAppConfigSwitch();
 
-            // Bug: there is no way to set activity type to W3C
-            // https://github.com/dotnet/runtime/issues/43853
```

Comment:
This bug is resolved:
- https://learn.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/5.0/default-activityidformat-changed